config/sidekiq.yml

```diff
@@ -0,0 +1,70 @@
+:schedule:
+  cmc_update_prices1_scheduler:
```

Comment:
It might make more sense to just have the scheduler run every minute but have the logic within the job handle what to update (via something like modulo division).

This way it reduces the amount of jobs in the queue, otherwise things can get a little crazy in Redis.